test/capture_test.lua

```diff
@@ -8,11 +8,14 @@ local capture = Capture:new()
 -- Disable luatest logging to avoid capturing it.
 require('luatest.log').info = function() end
 
-g.setup = function() capture:enable() end
-g.teardown = function()
```

Comment:
Can you add some stubs that would raise an error on an attempt to set `setup` or `teardown` hooks?